.specs/features/fase-2-consistencia-performance/tasks.md

```
@@ -0,0 +1,78 @@
+
+## Gate Check Commands
+
+| Gate Level | Command |
+|------------|---------|
+| Quick | `node_modules/.bin/vitest run <arquivos>` |
+| Full | `npm test` |
+| Build | `npm test && npm run lint && npm run build` |
+
+## Execution Plan
+
+### Phase A (Sequential): atômicos independentes
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)